Corona-Warn-App/src/test/java/de/rki/coronawarnapp/vaccination/core/server/valueset/VaccinationServerTest.kt

```diff
@@ -0,0 +1,109 @@
+package de.rki.coronawarnapp.vaccination.core.server.valueset
+
+import dagger.Lazy
+import de.rki.coronawarnapp.util.coroutine.DispatcherProvider
+import de.rki.coronawarnapp.util.security.SignatureValidation
+import de.rki.coronawarnapp.vaccination.core.server.valueset.internal.ValueSetInvalidSignatureException
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.just
+import io.mockk.runs
+import io.mockk.verify
+import okhttp3.Cache
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import testhelpers.BaseTest
+import java.io.File
+
+class VaccinationServerTest : BaseTest() {
+
+    /**
+     * contains both binary and signature
+     */
+    private val exportZip = File("src/test/resources/export.zip")
+
+    /**
+     * binary is missing
+     */
+    private val invalidExportZip = File("src/test/resources/export_invalid.zip")
```

Comment:
The names are a bit generic.
Maybe `resources/vaccination/valueset_(default|invalid).zip`?

Reply:
renamed it